src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js

```diff
@@ -48,8 +47,7 @@ export const getPodcastExternalLinks = async (
     if (!brandPid) return [];
     let downloadLink = [];
     const links = pathOr([], ['default', variant, brandPid], linkData);
-    // Burmese podcast experiment
-    if (service === 'burmese' && brandPid === 'p02pc9lh') {
+    if (service) {
```

Comment:
Think we may be able to remove this condition as this function should only run on services that have podcast pages.